templates/azuredisk-csi-driver.yaml

```diff
@@ -0,0 +1,24 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: ClusterAddon
+metadata:
+  name: azuredisk-csi-driver
+  labels:
+    kubeaddons.mesosphere.io/name: azuredisk-csi-driver
+    kubeaddons.mesosphere.io/provides: csi-driver
+  annotations:
+    appversion.kubeaddons.mesosphere.io/azuredisk-csi-driver: "0.4.0"
+    values.chart.helm.kubeaddons.mesosphere.io/azuredisk-csi-driver: "https://raw.githubusercontent.com/mesosphere/charts/94333f633ccb198574f55bf0618ea182c727657a/stable/azuredisk-csi-driver/values.yaml"
+spec:
+  kubernetes:
+    minSupportedVersion: v1.15.0
+  namespace: kube-system
+  cloudProvider:
+    - name: aws
+      enabled: false
```

Comment:
think you can remove this, as `aws` will never use this?

Reply:
I want to be as explicit as possible ;)